src/main/scala/com/fulcrumgenomics/umi/FilterConsensusReads.scala

```diff
@@ -96,8 +96,8 @@ class FilterConsensusReads
   val maxReadErrorRate: Seq[Double] = Seq(0.025),
   @arg(flag="e", minElements=1, maxElements=3, doc="The maximum error rate for a single consensus base.")
   val maxBaseErrorRate: Seq[Double] = Seq(0.1),
-  @arg(flag="N", minElements=1, maxElements=3, doc="Mask (make 'N') consensus bases with quality less than this threshold.")
-  val minBaseQuality: PhredScore = DefaultMinConsensusBaseQuality,
+  @arg(flag="N", doc="Mask (make 'N') consensus bases with quality less than this threshold.")
+  val minBaseQuality: PhredScore,
```

Comment:
@nh13 I'm actually tempted to go the other way and also remove the default for `--min-reads` in the filtering tool.  All the other filters that have defaults are not very depth dependent.  The error rates and %Ns should work equally well for consensus reads with 1 raw read as for ones with 50.  But `min-reads` and `min-base-quality` both are very dependent on the tag family size distribution.